region/maridia/inner-yellow.json

```diff
@@ -348,29 +424,53 @@
           "to": [
             {
               "id": 1,
-              "requires": [
-                {"or": [
-                  "Gravity",
-                  {"or": [
-                    {"and": [
-                      "canSuitlessMaridia",
-                      "HiJump"
-                    ]},
-                    {"and": [
-                      "canSuitlessMaridia",
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": ["Gravity"]
+                },
+                {
+                  "name": "Suitless Jump Assist",
+                  "notable": false,
+                  "requires": [
+                    "canSuitlessMaridia",
+                    {"or":[
+                      "HiJump",
                       "canSpringBallJumpMidAir"
                     ]}
-                  ]}
-                ]}
+                  ]
+                },
+                {
+                  "name": "Naked Watering Hole Escape",
+                  "notable": true,
+                  "requires": [
+                    "canSuitlessMaridia",
+                    "canStationarySpinJump"  
```

Comment:
`canSunkenDualWallClimb`?

Reply:
I'm not sure this is a tech. `canSunkenDualWallClimb` involves going back and forth between two walls two tiles apart. Here the walls are one tile apart and you just need to spam jump while facing in one direction.

If we determine it should be a tech, it would be a different IMO.